parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -571,7 +578,9 @@ printCase env im doc ms0 = PP.lines $ map each gridArrowsAligned where
   printGuard (Just (ABT.AbsN' _ g)) =
     -- strip off any Abs-chain around the guard, guard variables are rendered
     -- like any other variable, ex: case Foo x y | x < y -> ...
-    PP.group $ PP.spaced [(fmt S.DelimiterChar " |"), pretty0 env (ac 2 Normal im doc) g]
+    PP.group $
+      PP.indentAfterNewline "  " $
```

Comment:
Adds indentation in the case that the guard doesn't fit on the same line as the pattern.